src/analytics/private/sqls/redshift/refresh-mv-sp-status.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS {{schema}}.{{table_refresh_mv_sp_status}} (
```

Comment:
How many estimated records are to be inserted per workflow run? Will it be costly after running the workflow for a year?

Reply:
There are 13 SP, suppose we need to refresh 3 days data. For a app, there are 39 records every day.
For a year, there are 14235 records. The total record just several MB.